amplify.yml

```diff
@@ -20,6 +20,7 @@ frontend:
     preBuild:
       commands:
         - if [ "${AWS_BRANCH}" = "master" ]; then export REACT_APP_BACKEND_URL=${BACKEND_URL_PRODUCTION}; else export REACT_APP_BACKEND_URL=${BACKEND_URL_STAGING}; fi
+        - if [ "${AWS_BRANCH}" = "master" ]; then export REACT_APP_SENTRY_DSN=${SENTRY_DSN_PRODUCTION}; else export REACT_APP_SENTRY_DSN=${SENTRY_DSN_STAGING}; fi
```

Comment:
I need to include the Sentry DSN in the csp headers. This would require exposing `SENTRY_DSN_PRODUCTION` and `SENTRY_DSN_STAGING` as I was't able to use `${REACT_APP_SENTRY_DSN}` to the Content-Security-Policy headers (see https://github.com/aws-amplify/amplify-console/issues/644).

The Sentry DSN key isn't a secret so it isn't critical if it is exposed. (see https://forum.sentry.io/t/dsn-private-public/6297)